frontend/src/components/datasets/dataset-form/dataset-form.component.tsx

```diff
@@ -281,8 +282,9 @@ export function DatasetForm({ mode, modalCallbackOnSubmit, formRef, datasetId, d
         (namespace: string) => validateNamespace(namespace).unwrap(),
         [validateNamespace],
     );
-
-    const ownerIds = useGetDatasetOwnerIds(currentDataset?.id);
+    const datasetOwners = useGetDatasetOwnerIds(currentDataset?.id);
```

Comment:
In create mode there is no currentDataset ID right? Shouldn't this request be skipped in that case?